zipline/testing/fixtures.py

```diff
@@ -351,11 +355,13 @@ class WithAssetFinder(WithDefaultDateBounds):
     ASSET_FINDER_EQUITY_SIDS = ord('A'), ord('B'), ord('C')
     ASSET_FINDER_EQUITY_SYMBOLS = None
     ASSET_FINDER_EQUITY_NAMES = None
+    ASSET_FINDER_EQUITY_EXCHANGE = 'TEST'
     ASSET_FINDER_EQUITY_START_DATE = alias('START_DATE')
     ASSET_FINDER_EQUITY_END_DATE = alias('END_DATE')
+    ASSET_FINDER_COUNTRY_CODE = '??'
```

Comment:
Should this be `ASSET_FINDER_DEFAULT_COUNTRY_CODE`? I expect us to want to have more than one country code shortly, and it seems like this primarily configures the default.

Reply:
If you need more than one country you should just implement `make_exchange_info` just like when you need interesting assets and need to implement `make_equity_info`